DEPENDENCIES/ML-HElib/src/helayers/hebase/FileUtils.cpp

```diff
@@ -26,47 +26,87 @@
 
 #include "FileUtils.h"
 #include "boost/filesystem.hpp"
+#include <filesystem>
+#include <iomanip>
+#include <iostream>
 
 using namespace std;
```

Comment:
For code clarity and prevent ambiguity, I would recommend I would avoid `using namespace xxx` and explicitly call the functions using its fully qualified name, e.g `std::cout`.

Reply:
github issue will be created to revisit Namespace directive style choices.